assets/js/contact_us_form_element.ts

```diff
@@ -62,6 +71,10 @@ class ContactUsFormElement extends HTMLElement {
       : null;
   }
 
+  get unplannedOutage(): boolean {
```

Comment:
Nitpick: Similar to Ruby's `?` suffix, I find that an `is` prefix in JavaScript helps inform that the value is a boolean. Thoughts?

```suggestion
  get isUnplannedOutage(): boolean {
```

Reply:
Yup, that works